include/swift/AST/DiagnosticsSIL.def

```diff
@@ -402,9 +402,11 @@ NOTE(constexpr_unsupported_instruction_found, none,
 NOTE(constexpr_unsupported_instruction_found_here,none, "operation"
      "%select{| used by this call is}0 not supported by the evaluator", (bool))
 
-NOTE(constexpr_unknown_function_called, none,
-    "encountered call to '%0' whose body is not available", (StringRef))
-NOTE(constexpr_unknown_function_called_here, none,
+NOTE(constexpr_found_callee_with_no_body, none,
+    "encountered call to '%0' whose body is not available. "
+    "Imported functions must be marked @inlinable to constant evaluate.",
```

Comment:
Nit: attributes must be surrounded with single quotes. Might be worth adding a fix-it as well.

Reply:
Thanks @theblixguy